Libraries/CustomComponents/ListView/ListView.js

```diff
@@ -229,6 +223,14 @@ var ListView = React.createClass({
      */
     removeClippedSubviews: React.PropTypes.bool,
     /**
+     * Makes the sections headers sticky.  The sticky behavior means that it
+     * will scroll with the content at the top of the section until it reaches
+     * the top of the screen, at which point it will stick to the top until it
+     * is pushed off the screen by the next section header.
+     * @platform ios
+     */
+    stickySectionHeaders: React.PropTypes.bool,
```

Comment:
Thanks for the helpful insight, @nihgwu. It seems you knows more about this issue. I saw the last update on the followup PR has been made 28 days ago. Is there any work in progress?